### PR TITLE
Apply Styles to page

### PR DESCRIPTION
**Pull request by codedbystuart**
# Highlights 💯
Great work so far with the project @codedbystuart 
However, I spotted some changes that you might want to take a look at in order to make 
the code looks better.

almost_there

### Linters
I suggest you set up some Linters in your project, this will help you to detect any bad practices
while writing code and ensuring best practices.
Follow this [Link](https://github.com/microverseinc/linters-config/tree/master/html-css) for guidance on how to 
setup Linters in your project.

### Use section tag
Kindly, I also recommend that you use use a section tag instead of a div in this section on line 34.
```
  <div class="aboutus-area">
    <section id="contacts"></section>
    <section id="about">
      <iframe id="video" src="https://www.youtube.com/embed/dKq_xfCz3Jk"></iframe>
    </section>
    <section id="socials"></section>
  </div>
```
It helps so much with accessibility.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
+* {
```
Comment (Kamba56) [inline css/style.css:1]: - [ ] Please use inline comments to give your review. This project requires you to use GitHub features. This is an example of inline comments. Kindly check[ here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) on how to give inline comments.

- [Optional] I would suggest that you give all your comments as separate bullet points so they can be easily identified from one another.🙃

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @codedbystuart, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
AlmostSamGorskiGIF

## Highlights
- [x] Polite comments
- [x] Clear comments

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,40 +1,68 @@
...
+  </header>
+
+  <div class="form-area">
     <form action="submit" id="form">
       <input type="email" id="email" name="email" placeholder="Enter your email address">
       <input type="submit" id="submit">
     </form>
+  </div>
+
+  <div class="aboutus-area">
```
Comment (codedbystuart) [inline index.html:34]: Kindly, I also recommend that you use use a section tag instead of a div in this section.

**Review by codedbystuart — COMMENTED** [review]
# Highlights 💯
Great work so far with the project @codedbystuart 
However, I spotted some changes that you might want to take a look at in order to make 
the code looks better.

almost_there

## Required Changes ♻️
Check the comments under the review.

**Review by levy002 — APPROVED** [review]
## Hello @kaizipaul  :wave: 
### I'm Levy, your PR reviewer!!

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPREOVED :fire: :fire: 
## Highlights
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**